src/test/java/lotto/domain/LottoTest.java

```diff
@@ -17,29 +17,37 @@ public class LottoTest {
     @Test
     @DisplayName("생성자 의존성 주입 - 로또 번호")
     void constructorDI() {
-        assertThat(new Lotto(lottoNumbers()).lottoNumbers().size()).isEqualTo(45);
+        assertThat(new Lotto(LottoNumber.lottoNumbers().subList(0,6)).lottoNumbers().size()).isEqualTo(6);
```

Comment:
```suggestion
        assertThat(new Lotto(LottoNumber.lottoNumbers().subList(0, 6)).lottoNumbers()).hasSize(6);
```

1. 포맷팅도 항상 신경 써 주세요 😎
2. 그런데 이 테스트는 정확히 무엇을 검증하는 걸까요?